### DISCUSSION TITLE
Not showing on addons.mozilla.org when searching for Reddit

### DISCUSSION THREAD

**Discussion by matrixik** [OctoNezd/oldlander#82] [General]
Hi, juts want to give you info that your add-on is not visible when anyone search for "Reddit" or "old reddit" on Firefox add-ons page.

https://addons.mozilla.org/en-US/firefox/search/?q=Reddit&sort=updated&type=extension

**Comment by OctoNezd** [comment]
Submitted a rename on AMO, hope it will make it appear.
![image](https://github.com/OctoNezd/oldlander/assets/15308404/9309df67-1312-4076-aa9d-f51d54eca3a6)

**Comment by OctoNezd** [comment]
![image](https://github.com/OctoNezd/oldlander/assets/15308404/a7e438df-6925-4151-a3ea-3255ba332964)
Appears now 🎉

**Comment by matrixik** [comment]
Great, thank you.
